apps/web-giddh/src/app/ledger/components/updateLedgerEntryPanel/updateLedgerEntryPanel.component.scss

```diff
@@ -83,7 +83,7 @@ ng-select .select-drop i.fa.fa-caret-up::before {
     padding: 2px 5px !important;
 }
 
-.no-padding {
+.p-0 {
```

Comment:
Please check if this style is now required as we have global style for `padding: 0`